requirements.txt

```diff
@@ -340,8 +344,8 @@ six==1.15.0 \
     --hash=sha256:30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259 \
     --hash=sha256:8b74bedcbbbaca38ff6d7491d76f2b06b3592611af620f8426e82dddb04a5ced \
     # via gcloud, google-api-core, google-auth, google-cloud-bigquery, google-resumable-media, jsonschema, oauth2client, packaging, protobuf, pyrsistent, pytest-xdist, python-dateutil
-smart_open==1.10.0 \
-    --hash=sha256:bea5624c0c2e49987c227bdf3596573157eccd96fd1d53198856c8d53948fa2c \
+smart_open==1.11.1 \
+    --hash=sha256:221cc08ae926af6ad72d141f208d228e1e801b1ee9b15f3e466eecf89d931002 \
```

Comment:
I believe this is going to break uploading public data to GCS. I tried upgrading smart_open earlier and something changed with writing compressed data. I'll have to take a closer look before upgrading.